configuration.rst

```diff
@@ -176,12 +177,33 @@ server-unix-socket-mode
 
     server-unix-socket-mode = "660"
 
+.. _log-level:
+
+log-level
+---------
+
+  Specifies the level of information to be logged while running PostgREST.
+
+  .. code:: bash
+
+      # No requests are logged, only startup and db connection recovery messages are.
```

Comment:
```suggestion
      # Only startup and db connection recovery messages are logged, requests are not logged.
```

Reply:
Maybe, not mention the requests at all and leave it like this?

```suggestion
      # Only startup and db connection recovery messages are logged
```

Reply:
Yeah, agree.